<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSupplier_gDialog</class>
 <widget class="QDialog" name="CSupplier_gDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>141</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>141</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Папка заказчиков</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="labelName">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Наименование:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditName">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="checkBoxActual">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Актуальна</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QGridLayout" name="gLayoutInformation">
     <item row="1" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelUserD">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelUserI">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Пользователь</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelDateI">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelDateD">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="hLayoutOkCancel">
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
